bucket/forkgram.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "4.0.0",
+    "description": "Fork of Telegram Desktop messaging app.",
+    "homepage": "http://t.me/forkgram",
+    "license": {
+        "identifier": "GPL-3.0-with-OpenSSL-exception",
+        "url": "https://github.com/Forkgram/tdesktop/blob/dev/LICENSE"
+    },
+    "url": "https://github.com/Forkgram/tdesktop/releases/download/v4.0.0/Telegram.zip",
+    "hash": "sha1:e2a57ee049afb3d9dc78572c68c05a7a4cd8b205 ",
+    "persist": "tdata",
+    "bin": [
+        [
+            "Telegram.exe",
+            "Forkgram"
+        ]
+    ],
```

```suggestion
    "bin": [
        "Telegram.exe",
        [
            "Telegram.exe",
            "Forkgram"
        ],
    ],
```

I think some users will expect the file name to be `Telegram.exe` even if they are installing *Forkgram*. The better solution is to create shims in both names.